pkg/kn/commands/service/list_mock_test.go

```diff
@@ -82,6 +82,146 @@ func TestServiceListAllNamespaceMock(t *testing.T) {
 	r.Validate()
 }
 
+func TestListEmptyMock(t *testing.T) {
+	// New mock client
+	client := knclient.NewMockKnClient(t)
+
+	// Recording:
+	r := client.Recorder()
+
+	r.ListServices(knclient.Any(), &v1alpha1.ServiceList{}, nil)
+
+	output, err := executeServiceCommand(client, "list")
+	assert.NilError(t, err)
+	assert.Assert(t, util.ContainsAll(output, "No", "services", "found"))
+
+	r.Validate()
+}
+
+func TestGetEmptyMock(t *testing.T) {
+	// New mock client
+	client := knclient.NewMockKnClient(t)
+
+	// Recording:
+	r := client.Recorder()
+
+	r.ListServices(knclient.Any(), &v1alpha1.ServiceList{}, nil)
+
+	output, err := executeServiceCommand(client, "list", "bar")
+	assert.NilError(t, err)
+	assert.Assert(t, util.ContainsAll(output, "No", "services", "found"))
+
+	r.Validate()
+}
+
+func TestServiceListDefaultOutputMock(t *testing.T) {
+
+	// New mock client
+	client := knclient.NewMockKnClient(t)
+
+	// Recording:
+	r := client.Recorder()
+
+	service1 := createMockServiceWithParams("foo", "default", "http://foo.default.example.com", "foo-xyz")
+	service3 := createMockServiceWithParams("sss", "default", "http://sss.default.example.com", "sss-xyz")
+	service2 := createMockServiceWithParams("bar", "default", "http://bar.default.example.com", "bar-xyz")
+	serviceList := &v1alpha1.ServiceList{Items: []v1alpha1.Service{*service1, *service2, *service3}}
+	r.ListServices(knclient.Any(), serviceList, nil)
+
+	output, err := executeServiceCommand(client, "list")
+	assert.NilError(t, err)
+
+	outputLines := strings.Split(output, "\n")
+	assert.Check(t, util.ContainsAll(outputLines[0], "NAME", "URL", "LATEST", "AGE", "CONDITIONS", "READY", "REASON"))
+	assert.Check(t, util.ContainsAll(outputLines[1], "bar", "bar.default.example.com", "bar-xyz"))
+	assert.Check(t, util.ContainsAll(outputLines[2], "foo", "foo.default.example.com", "foo-xyz"))
+	assert.Check(t, util.ContainsAll(outputLines[3], "sss", "sss.default.example.com", "sss-xyz"))
+
+	r.Validate()
+}
+
+func TestServiceListAllNamespacesOutputMock(t *testing.T) {
```

Comment:
This seems redundant with https://github.com/knative/client/blob/master/pkg/kn/commands/service/list_mock_test.go#L31

Reply:
removed the redundant method